apps/kitty-krew/Dockerfile

```
@@ -0,0 +1,35 @@
+FROM oven/bun:latest as builder
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)